docs/devops.md

```diff
@@ -105,6 +105,7 @@ If you are creating a temporary domain, use the pattern `[namespace]-i2`. If cre
 9. Shell into the `arca-dc` server, and add the node to the cluster:
     - `microk8s add-node`
     - Copy the `microk8s join` command it generates.
+    - Use the `--worker` flag unless you specifically need the node to be a master node. New nodes should be workers by default.
```

Comment:
If the above command includes the `--worker` flag, then this note would be:
"If you need this node to be a master node, omit the `--worker` flag."